modules/serverless-logic-upgrade-deleting-migrating-workflows-with-preview-profile.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.34-to-1.35
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-deleting-migrating-workflows-with-preview-profile_{context}"]
+= Deleting and migrating workflows with the `preview` profile
```

Comment:
I don't think using backticks is OK in module headings - see guidance: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

I'm not exactly sure why, but I think it can cause issues with the display of headings.

Reply:
Okay, noted! I will remove the backticks